config/environments/production.rb

```diff
@@ -68,7 +68,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = true
+  config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
`Rails 5.2.2`以上ですので，こちらを変更する必要はありません。

```suggestion
  config.i18n.fallbacks = true
```